### ISSUE TITLE
hacktoberfest

### ISSUE THREAD

**Issue by softcreations01** [softcreations01/OAU-GDSC-CLOUD-Hacktoberfest_2023#1]

**Comment by AmosBlack** [comment]
what do we need to do ?

**Comment by softcreations01** [comment]
Contribute by adding a project to the coder or non-coder folder and add your details to the index.html file

**Comment by Kenil-kavar** [comment]
Just now i have added one of my project, Plz review and merge it

**Comment by Ayon1308** [comment]
Well , I am new to this , so can anyone direct me what to do??

**Comment by softcreations01** [comment]
Follow the instructions in the readme file directly

**Comment by Ayon1308** [comment]
ok @softcreations01

**Comment by Shashi3k** [comment]
Is this issue still open. If yes, can u assign it to me

**Comment by elsa-linu** [comment]
Is this issue still available ? I'm happy to contribute.

**Comment by Shashi3k** [comment]
Hey i have raised a pull request. Please accept it

**Comment by softcreations01** [comment]
> Hey i have raised a pull request. Please accept it

Accepted

**Comment by Kenil-kavar** [comment]
Plz review check my pull request also

**Comment by mayankm1004** [comment]
If this issue still available I would like to contribute!!!

**Comment by harshit1142** [comment]
Please /assign this to me.

**Comment by elsa-linu** [comment]
Hi @softcreations01, thank you for assigning me. Please give me a day more as I'm new to it, I will raise a PR tomorrow. Thanks you.

**Comment by armaf002** [comment]
> Contribute by adding a project to the coder or non-coder folder and add your details to the index.html file

Are we to just add any project

**Comment by softcreations01** [comment]
> > Contribute by adding a project to the coder or non-coder folder and add your details to the index.html file
> 
> Are we to just add any project

Yes, it's an open source project mainly for learning and collaborating together widely around the globe

**Comment by softcreations01** [comment]
> Hi @softcreations01, thank you for assigning me. Please give me a day more as I'm new to it, I will raise a PR tomorrow. Thanks you.

You're welcome

**Comment by BinaryBhakti** [comment]
Is the issue still open? I would like to contribute.

**Comment by softcreations01** [comment]
Ye

> Is the issue still open? I would like to contribute.

Yes you can

**Comment by SanikaBhalerao1345** [comment]
hi! I'm also interested in contribution. is the issue still open

**Comment by softcreations01** [comment]
Alright,  I'll be glad to have you contribute to the project

**Comment by INVOKER2002** [comment]
hi there just came across your repo , would love to chat with you guys and contribute to this repo

**Comment by INVOKER2002** [comment]
can i add my projects in any language ? or does it have to be web development

**Comment by softcreations01** [comment]
> can i add my projects in any language ? or does it have to be web development

Any language works, add in the appropriate folder
